helm/fluent-logshipping-app/.kube-linter.yaml

```diff
@@ -0,0 +1,4 @@
+checks:
+  addAllBuiltIn: true
+  exclude:
+    - *
```

Comment:
please consider actually listing the tests you need to skip. The linter is there for a reason.

Reply:
done